xml/docu_styleguide.smartdocs.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       xmlns:its="http://www.w3.org/2005/11/its" version="5.0" xml:id="sec-smartdocs">
+
+ <title>Working with &sd;</title>
+ <para>
+  &sd; represent the modular approach to documentation whose goal is producing sets of
+  adaptable, solution-based, easy-to-find and reusable documents.
+ </para>
+ <para>
+  &sd; consist of smaller information units with the following characteristics:
+ </para>
+<itemizedlist>
+  <listitem>
+   <formalpara>
+    <title>Self-contained</title>
+    <para>
+     Each information unit stands for itself, including requirements, context, instructions, troubleshooting and FAQs.
```

Comment:
IHMO this is not correct. Units are not completely self contained, e.g. not everything includes requirements, hardly anything contains troubleshooting instructions, and glue topics are probably not even meant to be standalone.
We should be more careful with the terminology here with unit (concept, task, reference, glue) on the one hand and the topic on the other.

Reply:
Hmmm, and now it get's murky ... . In the doc-modular repo we use topic as you would use "unit". The "self-contained" statement is a bit dangerous. The assembly in itself has to be self-contained, but ideally all the topics building up the assembly/article, should also aim to be self-contained and not be dependent on other topics. This should also be fixed in the README of doc-modular where this was taken from. Let's discuss ;)

Reply:
So we agreed on the following:

* Follow the DITA spec and only use the term 'topic' for the topic as information units (topic types etc.), not for topic as in 'subject'.
* Topics are the building blocks for articles.
* Minimize the use of 'unit', only use it where necessary.

I tried to summarize all of this in 5319f6463f8c30eb9fde15817b6d776bb8d162e6. Please let me know what you think.